workbench-core/example/infrastructure/integration-tests/tests/users/getUserRoles.test.ts

```diff
@@ -36,14 +36,9 @@ describe('userManagement get user roles integration test', () => {
   });
 
   it('returns a list containing the users roles', async () => {
-    const roleName = `example-role-name-${uuidv4()}`; // TODO use roles.create() response once the route returns the created role name
     const { data: userData } = await adminSession.resources.users.create(user);
-    await adminSession.resources.roles.create({ roleName });
-    await adminSession.resources.roles.role(roleName).addUser({ userId: userData.id });
-
     const { data } = await adminSession.resources.users.user(userData.id).getRoles();
-
-    expect(data).toMatchObject({});
```

Comment:
why remove the UserManagement component test?

Reply:
I completely forgot a separate router is used in the example app :facepalm:. Replacing that now (but will keep the expectation change).